Assets/MixedRealityToolkit.Tests/EditModeTests/Core/TestFixture_02_MixedRealityToolkitFilesTests.cs

```diff
@@ -0,0 +1,121 @@
+                {
+                    TestValidPath($"MixedRealityToolkit.{moduleNames[i]}", module);
+                }
+            }
+        }
+
+        [Test]
+        public void Test_02_FindMatchingModuleNuget()
+        {
+            // Test invalid base name
+            TestInvalidPath("aaa.1.23-45/MRTK");
+            TestInvalidPath(".1.23-45.SDK/MRTK");
+            TestInvalidPath("aaa.1.23-45.SDK/MRTK");
+
+            // Test missing chars
+            TestInvalidPath("Microsoft.MixedReality.Toolki.SDK.1.23-45/MRTK");
+            TestInvalidPath("Microsoft.MixedReality.Toolkit.SD.1.23-45/MRTK");
+            TestInvalidPath("Microsoft.MixedReality.Toolkit.SDK.1.23-45/MRT");
+
+            TestInvalidPath("icrosoft.MixedReality.Toolkit.SDK.1.23-45/MRTK");
+            TestInvalidPath("Microsoft.MixedReality.Toolkit.DK.1.23-45/MRTK");
+            TestInvalidPath("Microsoft.MixedReality.Toolkit.SDK.1.23-45/RTK");
+
+            // Test missing dots
+            TestInvalidPath("Microsoft.MixedReality.Toolkit.SDK1.23-45/MRTK");
+            TestInvalidPath("Microsoft.MixedReality.ToolkitSDK.1.23-45/MRTK");
+            TestInvalidPath("Microsoft.MixedReality.ToolkitSDK1.23-45/MRTK");
+
+            // Test missing version
+            TestInvalidPath("Microsoft.MixedReality.Toolkit.SDK/MRTK");
+            // Test missing MRTK suffix
+            TestInvalidPath("Microsoft.MixedReality.Toolkit.SDK.1.23-45");
+
+            // Test valid paths
+            TestValidPath("Microsoft.MixedReality.Toolkit.1.23-45/MRTK", MixedRealityToolkitModuleType.Core);
+            TestValidPath("Microsoft.MixedReality.Toolkit.SDK.1.23-45/MRTK", MixedRealityToolkitModuleType.SDK);
+
+            var modules = Enum.GetValues(typeof(MixedRealityToolkitModuleType));
+            var moduleNames = Enum.GetNames(typeof(MixedRealityToolkitModuleType));
+            for (int i = 0; i < modules.Length; ++i)
+            {
+                var module = (MixedRealityToolkitModuleType)modules.GetValue(i);
+                if (module != MixedRealityToolkitModuleType.Core)
+                {
+                    TestValidPath($"Microsoft.MixedReality.Toolkit.{moduleNames[i]}.1.23-45/MRTK", (MixedRealityToolkitModuleType)modules.GetValue(i));
+                }
+            }
+        }
+
+        public void TestInvalidPath(string path)
+        {
+            Assert.False(MixedRealityToolkitFiles.FindMatchingModule(path, out MixedRealityToolkitModuleType module));
+        }
+
+        public void TestValidPath(string path, MixedRealityToolkitModuleType expectedModule)
+        {
+            Assert.True(MixedRealityToolkitFiles.FindMatchingModule(path, out MixedRealityToolkitModuleType module));
+            Assert.AreEqual(module, expectedModule);
+        }
+
+        [TearDown]
+        public void CleanupTests()
+        {
+            TestUtilities.TearDownScenes();
+        }
+
+        private void WaitUntilFoldersAvailable()
```

Comment:
I think this may be dead right? I assume this was from maybe some earlier iteration that tried to do real file IO stuff